third_party/3/six/__init__.pyi

```diff
@@ -13,6 +12,7 @@ from typing import (
     Dict,
     ItemsView,
     Iterable,
+    Iterator,
```

Comment:
I don't think removing `Iterator` from `six` is correct. It is explicitly defined in the implementation and not identical to `typing.Iterator`.

Reply:
Good catch. I've backed out the change from this stub file.